### PR TITLE
[React] - Updated naming convention

### PR DESCRIPTION
**Pull request by juampick**

### CODE REVIEW

**Comment by matiasbeckerle** [comment]
I would like to hear an explanation or the reasoning behind your proposal @juampick (and every proposal in the future of everyone).

I too prefer `js` over `jsx` in order to avoid a small "customization" at pipeline process (take into account jsx extension) but also doesn't provide me any visual benefit or speedup for development purposes.

Maybe someone who wants to keep the jsx just for the sake of a wide adopted convention in React community? I'm all ears!

File: React/README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/react),
 
 ## Naming
 
-  - **Extensions**: Use `.jsx` extension for React components.
+  - **Extensions**: Use `.js` extension for React components.
   - **Filename**: Use PascalCase for filenames. E.g., `ReservationCard.jsx`.
```
Comment (matiasbeckerle) [inline React/README.md]: I guess you will need to update this line in order to match the change from above.

Comment (juampick) [inline React/README.md]: Exactly, good catch

**Comment by juampick** [comment]
Hi @matiasbeckerle good question!

I do not have an explanation (I think there is no one) between using `jsx` or `js` extension. 
It's up to the developer, and in the react world (as many many things), there are some that choose either one or the other ;) - you have to choose in this case.

But for example, I may recommend using `js` because of that *facebook* uses on their starter kit: 
https://github.com/facebookincubator/create-react-app.

There are some good links here: 
https://github.com/facebook/react/issues/3582#issuecomment-89411479
and here: https://github.com/kriasoft/react-starter-kit/issues/52

But as some guy in github comments: summarizing is: 
`jsx is recommended by AirBnb guys, .js by Facebook guys.`.. I would prefer follow Facebook.

I do not see any advantage of using .jsx. One of them could be that the IDE can recognize and compile those files to `js` but, you are using automatically babel so, you don't need to do that conversion manually.

Thanks for your comment, I hope this clarifies a bit more,
JC

**Comment by matiasbeckerle** [comment]
For sure that covers my question, thank you very much for the insights @juampick!

**Comment by juampick** [comment]
That's nice @matiasbeckerle 🗡  thanks for reading this ;)

**Comment by AlphaGit** [comment]
Thanks @matiasbeckerle -- I don't have any particular opinion on reason on this change, but I think we should start being very skeptic about changes and really asking about them how you did. Specially if we're going to turn these into "guidelines", I think it's very important that we decide ourselves what they should be.

And a totally valid guideline is "we have no preference, we let the developer decide". Being precise does not require being strict. :)

Thanks @juampick for the changes and @matiasbeckerle for the questions!

**Comment by juampick** [comment]
Good point @AlphaGit regarding strict way ;) thanks

**Comment by matiasbeckerle** [comment]
I think I don't get the whole comment @AlphaGit 😄  sorry -- but the quote...

> we have no preference, we let the developer decide

...could be on the front page!

**Pull request merged**